Modelica/package.mo

```diff
@@ -2492,6 +2504,9 @@ that can lead to wrong simulation results):
 <tr><td colspan=\"2\"><strong>Modelica.Blocks.Math</strong></td></tr>
 <tr><td>Pythagoras</td>
     <td>The case with negative <code>y2</code> was not correctly considered if <code>u1IsHypotenuse</code> is <code>true</code>. This has been corrected.</td></tr>
+<tr><td colspan=\"2\"><strong>Modelica.Electrical.Semiconductors</strong></td></tr>
+<tr><td>Diode</td>
+    <td>Fix bug in current equations.</td></tr>
```

Comment:
```suggestion
    <td>Fixed unit error in current equations.</td></tr>
```

I believe you're refering to https://github.com/modelica/ModelicaStandardLibrary/pull/3010#pullrequestreview-301912956.

Reply:
Yes, particularly https://github.com/modelica/ModelicaStandardLibrary/pull/3010/commits/3f23a7227142ba6a2161aba640a08792f7c7cc9a